dataprofiler/profilers/profile_builder.py

```diff
@@ -402,7 +402,11 @@ def load_from_dict(cls, data, options: dict | None = None) -> StructuredColProfi
             if attr == "options" and value is not None:
                 value = load_option(value, options)
             if attr == "_null_values":
-                value = {k: profile._null_values[k] for k, v in value.items()}
```

Comment:
this is a bug and would cause errors if the prev values didn't exist in the default.